### PR TITLE
New dialogs

### PR DESCRIPTION
**Pull request by maxwellfundi**

### CODE REVIEW

File: instat/dlgGroup.Designer.vb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+﻿<Global.Microsoft.VisualBasic.CompilerServices.DesignerGenerated()> _
+Partial Class dlgGroup
```
Comment (dannyparsons) [inline instat/dlgGroup.Designer.vb]: Can this have a more specific name? We might get confused later with this name.

File: instat/dlgMultipleRegression.vb [hunk 2]
```diff
@@ -16,21 +16,37 @@
...
+
+    Private Sub InitialiseDialog()
+
     End Sub
 
-    Private Sub defaultSettings()
-        ucrAddRemove.lstAvailableVariable.ResetText()
-        ucrReceiveExplanatoryVaraibles.lstSelectedVariables.Items.Clear()
-        ucrReceiveResponseVariable.ResetText()
+    Private Sub Setdefaults()
```
Comment (dannyparsons) [inline instat/dlgMultipleRegression.vb]: Capital D needed for Defaults

File: instat/dlgGeneralTables.vb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        End If
+        autoTranslate(Me)
+        TestOkEnabled()
+    End Sub
+
+    Private Sub InitialiseDialog()
+
+    End Sub
+
+    Private Sub Setdefaults()
```
Comment (dannyparsons) [inline instat/dlgGeneralTables.vb]: Capital D needed for Defaults

File: instat/dlgGoodnessofFit.vb [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        End If
+        autoTranslate(Me)
+        TestOkEnabled()
+    End Sub
+
+    Private Sub InitialiseDialog()
+
+    End Sub
+
+    Private Sub Setdefaults()
```
Comment (dannyparsons) [inline instat/dlgGoodnessofFit.vb]: And here

File: instat/dlgGroup.vb [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        End If
+        autoTranslate(Me)
+        TestOkEnabled()
+    End Sub
+
+    Private Sub InitialiseDialog()
+
+    End Sub
+
+    Private Sub Setdefaults()
```
Comment (dannyparsons) [inline instat/dlgGroup.vb]: And here

File: instat/dlgMultipleResponse.vb [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+        End If
+        autoTranslate(Me)
+        TestOkEnabled()
+    End Sub
+
+    Private Sub InitialiseDialog()
+
+    End Sub
+
+    Private Sub Setdefaults()
```
Comment (dannyparsons) [inline instat/dlgMultipleResponse.vb]: And here

File: instat/dlgOneSampleSummaryData.vb [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+        End If
+        autoTranslate(Me)
+        TestOkEnabled()
+    End Sub
+
+    Private Sub InitialiseDialog()
+
+    End Sub
+
+    Private Sub Setdefaults()
```
Comment (dannyparsons) [inline instat/dlgOneSampleSummaryData.vb]: And here

File: instat/dlgTwoSampleSummaryData.vb [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+        End If
+        autoTranslate(Me)
+        TestOkEnabled()
+    End Sub
+
+    Private Sub InitialiseDialog()
+
+    End Sub
+
+    Private Sub Setdefaults()
```
Comment (dannyparsons) [inline instat/dlgTwoSampleSummaryData.vb]: And here

File: instat/dlgWindrose.vb [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+        End If
+        autoTranslate(Me)
+        TestOkEnabled()
+    End Sub
+
+    Private Sub InitialiseDialog()
+
+    End Sub
+
+    Private Sub Setdefaults()
```
Comment (dannyparsons) [inline instat/dlgWindrose.vb]: And here

**Comment by maxwellfundi** [comment]
@dannyparsons this is now fixed

**Pull request merged**